examples/webpack1/.babelrc

```diff
@@ -5,7 +5,6 @@
     "stage-1"
   ],
   "plugins": [
-    "lodash",
-    ["lodash", { "id": "semantic-ui-react" }]
+    ["lodash", { "id": ["lodash", "semantic-ui-react"] }]
```

Comment:
We [can](https://github.com/lodash/babel-plugin-lodash#usage) run resolution updates in single run.